roles.md

```diff
@@ -105,6 +105,15 @@ The scientific advisory boards consists of highly experienced scientists devoted
 
 Members of the scientific adivsory boards are nominated by the core team. 
 
+## NumFOCUS subcommittee
+
+The core team will maintain a subcommittee to manage its interactions with NumFOCUS.
+
+* The NumFOCUS Subcommittee is comprised of 5 persons who manage project funding that comes through NumFOCUS. It is expected that these funds will be spent in a manner that is consistent with the non-profit mission of NumFOCUS and the direction of the Project as determined by the full Council.
```

Comment:
This does NOT need to be restricted to 5 people. IIRC it was 5+. I know that a limit of 5 people can speed up some processes but I think that it's pretty difficult to always justify who shall be in the subcommittee and who is not (even though we don't grant these people additional power).

` the full Council`

This would mean only Isaac, Danila + me? I thought that we had the final say but not the ruling power. Maybe rephrase this to core team?

Reply:
agree this should read "core team"

Reply:
```suggestion
* The NumFOCUS Subcommittee is comprised of at least 5 persons who manage project funding that comes through NumFOCUS. It is expected that these funds will be spent in a manner that is consistent with the non-profit mission of NumFOCUS and the direction of the Project as determined by the Core Team.
```